runtime/themes/nightfox.toml

```diff
@@ -0,0 +1,197 @@
+
+"property" = { fg = "blue" } # Regex group names.
+"special" = { fg = "fg2"} # Special symbols e.g `?` in Rust, `...` in Hare.
+"attribute" = { fg = "yellow" } # Class attributes, html tag attributes.
+
+"type" = { fg = "yellow"} # Variable type, like integer or string, including program defined classes, structs etc..
+"type.builtin" = { fg = "cyan-bright" } # Primitive types of the language (string, int, float).
+"type.enum.variant" = { fg = "orange" }
+
+"constructor" = { fg = "magenta"} # Constructor method for a class or struct.
+
+"constant" = { fg = "yellow" } # Constant value
+"constant.builtin" = { fg = "orange-bright" } # Special constants like `true`, `false`, `none`, etc.
+"constant.builtin.boolean" = { fg = "orange" } # True or False.
+"constant.character" = { fg = "green" } # Constant of character type.
+"constant.character.escape" = { bold = true, fg = "yellow-bright" } # escape codes like \n.
+"constant.numeric"  = { fg = "orange" } # constant integer or float value.
+"constant.numeric.integer" = { fg = "orange" } # constant integer value.
+"constant.numeric.float" = { fg = "orange" } # constant float value.
+
+"string" = { fg = "green" } # String literal.
+"string.regexp" = { fg = "yellow-bright" } # Regular expression literal.
+"string.special" = { fg = "yellow-bright", modifiers = ["bold"] } # Strings containing a path, URL, etc.
+"string.special.url" = { bold = true, fg = "cyan" } # String containing a web URL.
+# "string.special.path" = { } # String containing a file path.
+# "string.special.symbol" = { } # Erlang/Elixir atoms, Ruby symbols, Clojure keywords.
+
+"comment" = { fg = "comment" } # This is a comment.
+# "comment.line" = { } # Line comments, like this.
+# "comment.block" = { } # Block comments, like /* this */ in some languages.
+# "comment.block.documentation" = { } # Doc comments, e.g '///' in rust.
+
+"variable" = { fg = "white" }  # Variable names.
+"variable.builtin" = { fg = "red" } # Language reserved variables: `this`, `self`, `super`, etc.
+"variable.parameter" = { fg = "cyan-bright" } # Function parameters.
+# "variable.other" = { }
+"variable.other.member" = { fg = "fg2" } # Fields of composite data types (e.g. structs, unions).
+
+"label" = { fg = "magenta-bright" } # Loop labels, among other things.
+
+"punctuation" = { fg = "fg2" } # Any punctuation symbol.
+"punctuation.delimiter" = { fg = "fg2" } # Commas, colons or other delimiter depending on the language.
+"punctuation.bracket" = { fg = "fg2" } # Parentheses, angle brackets, etc.
+"punctuation.special" = { fg = "fg2" } # String interpolation brackets
+
+"keyword" = { fg = "magenta" } # Language reserved keywords.
+"keyword.control" = { fg = "magenta-bright" } # Control keywords.
+"keyword.control.conditional" = { fg = "magenta-bright" } # `if`, `else`, `elif`.
+"keyword.control.repeat" = { fg = "magenta-bright" } # `for`, `while`, `loop`.
+"keyword.control.import" = { fg = "magenta-bright" } # `import`, `export` `use`.
```

Comment:
You don't need all these additional definitions, `keyword.control.conditional` will fallback onto `keyword.control` and so on